spec/rb_lib_text_spec.rb

```diff
@@ -10,4 +10,19 @@
     tokens = RbLibText.tokens(sentance)
     expect(tokens.count).to eq(16)
   end
+
+  it 'keeps hearts intact' do
+    sentence = "My first open source PR! <3 #itwasfun"
+    tokens = RbLibText.tokens(sentence)
+    expect(tokens).to eq(["My", "first", "open", "source", "PR", "!", "<3", "#itwasfun"])
+    expect(tokens.count).to eq(8)
```

Comment:
you shouldn't need to check the length of the list after you check its contents, should you?